test/unit-tests/MultiBridgeMessageSender.t.sol

```diff
@@ -366,14 +357,77 @@ contract MultiBridgeMessageSenderTest is Setup {
         vm.startPrank(caller);
 
         address[] memory senderAdapters = new address[](3);
-        senderAdapters[0] = failingAdapterAddr;
-        senderAdapters[1] = axelarAdapterAddr;
-        senderAdapters[2] = wormholeAdapterAddr;
-
+        uint256[] memory fees = new uint256[](3);
         bool[] memory adapterSuccess = new bool[](3);
-        adapterSuccess[0] = false;
-        adapterSuccess[1] = true;
-        adapterSuccess[2] = true;
+        (uint256 wormholeFee,) = IWormholeRelayer(POLYGON_RELAYER).quoteEVMDeliveryPrice(
```

Comment:
I'd suggest using the existing two-array sorter for three arrays, in place of the lengthy if statement. the function could look as follows

```
    function _sortThreeAdaptersWithFeesAndOps(address[] memory _adapters, uint256[] memory _fees, bool[] memory _ops)
        internal
        pure
        returns (address[] memory sortedAdapters, uint256[] memory sortedFees, bool[] memory sortedOps)
    {
        require(_adapters.length == 3 && _fees.length == 3 && _ops.length == 3, "invalid params");

        sortedAdapters = new address[](3);
        sortedFees = new uint256[](3);
        sortedOps = new bool[](3);
        (address[] memory s1Ads, uint256[] memory s1Fees, bool[] memory s1Ops) =
            _sortTwoAdaptersWithFeesAndOps(_adapters[0], _adapters[1], _fees[0], _fees[1], _ops[0], _ops[1]);
        if (_adapters[2] > s1Ads[1]) {
            (sortedAdapters[0], sortedAdapters[1], sortedAdapters[2]) = (s1Ads[0], s1Ads[1], _adapters[2]);
            (sortedFees[0], sortedFees[1], sortedFees[2]) = (s1Fees[0], s1Fees[1], _fees[2]);
            (sortedOps[0], sortedOps[1], sortedOps[2]) = (s1Ops[0], s1Ops[1], _ops[2]);
        } else {
            (address[] memory s2Ads, uint256[] memory s2Fees, bool[] memory s2Ops) =
                _sortTwoAdaptersWithFeesAndOps(_adapters[2], s1Ads[0], _fees[2], s1Fees[0], _ops[2], s1Ops[0]);
            (sortedAdapters[0], sortedAdapters[1], sortedAdapters[2]) = (s2Ads[0], s2Ads[1], s1Ads[1]);
            (sortedFees[0], sortedFees[1], sortedFees[2]) = (s2Fees[0], s2Fees[1], s1Fees[1]);
            (sortedOps[0], sortedOps[1], sortedOps[2]) = (s2Ops[0], s2Ops[1], s1Ops[1]);
        }
    }
```

Reply:
Thanks. This does look cleaner.